sycl/doc/extensions/supported/sycl_ext_intel_esimd/examples/README.md

```diff
@@ -65,4 +65,102 @@ is to show the basic ESIMD APIs in well known examples.
           });
     });
     ```
-3) TODO: Add more examples here.
+3) Dot Product Accumulate Systolic (DPAS) API - ["dpas"](./dpas.md)
+
+   Please see the full source code here: ["dpas"](./dpas.md)
+
+   ```c++
+    // Res = A * B.
+    // Assume the HW is PVC.
+
+    constexpr int SystolicDepth = 8;
+    constexpr int RepeatCount = 4;
+    constexpr int ExecSize = 16; // 16 for PVC, 8 for DG2.
+
+    // Let A and B be matrices of unsigned 4-bit integers.
+    constexpr xmx::dpas_argument_type BPrec = xmx::dpas_argument_type::u4;
+    constexpr xmx::dpas_argument_type APrec = xmx::dpas_argument_type::u4;
+
+    constexpr int AElemBitSize = 4; // 4-bit integers.
+    constexpr int BElemBitSize = 4; // 4-bit integers.
+
+    // Elements of A and B will are packed into uint8_t,
+    // meaning that one uint8_t holds two 4-bit unsigned integers.
+    // Packaging for A and res is horizontal, for B is vertical.
+    using PackedType = unsigned char;
+    using APackedType = PackedType;
+    using BPackedType = PackedType;
+
+    // res type, according to documentation is either int or uint.
```

Comment:
```suggestion
    // result type, according to documentation is either int or uint.
```

just making it a little more clear, it took me a quick second to figure it out

Reply:
Initially I was referring to variable `res`, but it is much better just to say `Result`.